ext/background/filter/github_enterprise.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+registerFilter({
+  url_regex: [/(.+)github\.com/, /github(.+)\.com/],
```

Comment:
This feels fairly brittle, but it's hard to know what to match on because it'll be sitting on a custom domain...

Maybe in the future we could allow for this to be user-configurable somehow?

Thoughts @dpup